### DISCUSSION TITLE
WebDAV复制文件时,提示无法复制 ***。抱歉，此用户帐户权限不足，无法执行此操作。

### DISCUSSION THREAD

**Discussion by cdphantom** [AlistGo/alist#5220] [Q&A]
### Please make sure of the following things

- [X] I have read the [documentation](https://alist.nn.ci).
我已经阅读了[文档](https://alist.nn.ci)。

- [X] I'm sure there are no duplicate issues or discussions.
我确定没有重复的issue或讨论。

- [X] I'm sure it's due to `AList` and not something else(such as [Network](https://alist.nn.ci/faq/howto.html#tls-handshake-timeout-read-connection-reset-by-peer-dns-lookup-failed-connect-connection-refused-client-timeout-exceeded-while-awaiting-headers-no-such-host) ,`Dependencies` or `Operational`).
我确定是`AList`的问题，而不是其他原因（例如[网络](https://alist.nn.ci/zh/faq/howto.html#tls-handshake-timeout-read-connection-reset-by-peer-dns-lookup-failed-connect-connection-refused-client-timeout-exceeded-while-awaiting-headers-no-such-host)，`依赖`或`操作`）。

- [X] I'm sure this issue is not fixed in the latest version.
我确定这个问题在最新版本中没有被修复。

### AList Version / AList 版本

v3.27.0

### Driver used / 使用的存储驱动

阿里云盘 Open

### Describe the bug / 问题描述

存储用的阿里云盘 Open,状态正常,可正常下载文件,速度也很快,30Mbps
我的群晖NAS上挂在了 WebDAV,复制文件到阿里云盘 Open时,报错:无法复制 ***.mp4。抱歉，此用户帐户权限不足，无法执行此操作。
OAuth验证和令牌已经按最新的设置了, 是哪里设置的有问题吗?

### Reproduction / 复现链接

抱歉,图片无法上传

### Config / 配置

抱歉,图片无法上传

### Logs / 日志

无报错日志

**Comment by welcome** [comment]
Thanks for opening your first issue here! Be sure to follow the issue template!

**Comment by Viewday** [comment]
我也是这个问题，一直搜不到解决方法，其他网盘正常，就阿里云盘有问题

**Comment by zyystudio** [comment]
同问，我百度网盘也是这个问题但是阿里云盘没有问题。而且我发现百度网盘对于旧文件比如半年前的会有这个问题试了几个转存的新电影文件夹都没报错很奇怪。

**Comment by chenxuanjie** [comment]
我用的CloudDriver2挂载的，在权限处的0755改成0777就可以复制和移动了

**Comment by baofaguo** [comment]
same here
